tailcall-aws-lambda/Cargo.toml

```diff
@@ -24,6 +24,6 @@ dotenvy = "0.15.7"
 
 anyhow = "1.0.82"
 async-trait = "0.1.80"
-hyper = { version = "0.14.28", default-features = false }
-reqwest = { version = "0.11", default-features = false }
+hyper = { version = "1.3.1", default-features = false }
```

Comment:
hyper needs a lot of changes.. I'll be raising a PR soon (I had a pr for that #1241 but those were a lot of changes so it didn't get merged back then)

Reply:
I have reverted this change. it wasn't required to fix this issue